react-form-controls/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+export default function RegistrationFormUncontrolled() {
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
```

Comment:
I prefer to import `FormEvent` from `react` so I don't have to prefix it with `React.`. But it's just preference.